docs/markdown/get_started/developers/contributing/development_process.md

```diff
@@ -178,7 +178,10 @@ git push -f origin HEAD
 
 - After a Gestalt maintainer adds a correct semver label and approves a Pull Request, the PR will be ready to merge. Coordinate with the reviewer to determine when the PR should be merged.
 
-- Check the status of your PR https://github.com/pinterest/gestalt/pull/XXXX and access the deploy preview using the built site URL https://deploy-preview-XXXX--gestalt.netlify.app from Netlify.
+- Check the status of your PR https://github.com/pinterest/gestalt/pull/XXXX and access the deploy preview using the built site URL https://deploy-preview-XXXX--gestalt.netlify.app from Netlify. 
+
+- When sharing development urls, you may have to remind viewers to toggle on development mode from the header menu. To simplify sharing, you can add a `?devexample=true` parameter to the docs url to enable the examples by default.
```

Comment:
This should also include a reason why dev mode needs to be turned on, something like
"…to toggle on development mode from the header menu. This is necessary to see unpublished changes in the docs code examples. To simplify sharing…"

Reply:
done